sycl/plugins/unified_runtime/ur/adapters/level_zero/common.cpp

```diff
@@ -285,3 +285,8 @@ ur_result_t zerPluginGetLastError(char **message) {
   *message = &ErrorMessage[0];
   return ErrorMessageCode;
 }
+
+void sycl::detail::ur::die(const char *Message) {
+  std::cerr << "ur_die: " << Message << std::endl;
+  std::terminate();
+}
```

Comment:
Using the `sycl` namespace in the adapter files will likely cause out-of-tree build failures similar to the CUDA adapter issue that has a fix in https://github.com/intel/llvm/pull/10135

Reply:
OK I've removed the pi::die impl that I added to level_zero. I think it is best if level_zero folk deal with this in the way they want anyway.